addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -108,7 +109,8 @@ def _compute_l10n_latam_amount_and_taxes(self):
                     sign = -1
                 else:
                     sign = 1
-                l10n_latam_amount_untaxed = invoice.amount_untaxed + sign * sum(included_invoice_taxes.mapped('balance'))
+                amount = 'amount_currency' if len(currencies) == 1 else 'balance'
+                l10n_latam_amount_untaxed = invoice.amount_untaxed + sign * sum(included_invoice_taxes.mapped(amount))
```

Comment:
could be amount in all cases then?

Reply:
i think it's not necessary, because the amount of the taxes is only used in this code line.